SDK_dotnet/Clients/QRCodeClient.cs

```diff
@@ -1,166 +1,166 @@
-﻿using Gluwa.SDK_dotnet.Error;
```

Comment:
It seems like your line ending changed for this file. revert the line ending as it was before. Otherwise, I can't really tell what you've changed